### ISSUE TITLE
Add support for Eclipse 4.25 (2022-09)

### ISSUE THREAD

**Issue by eric-milles** [groovy/groovy-eclipse#1377]
https://www.eclipse.org/eclipse/news/4.25

**Comment by mpiggott** [comment]
I think the version ranges for m2e may be wrong for what was shipped with 4.25, when installing (into the committers bundle):

```
Cannot complete the install because one or more required items could not be found.
  Software being installed: Groovy-Eclipse M2E integration 4.7.0.v202209301526-e2209-RELEASE (org.codehaus.groovy.m2eclipse.feature.feature.group 4.7.0.v202209301526-e2209-RELEASE)
  Missing requirement: Groovy m2e integration 4.7.0.v202209301526-e2209-RELEASE (org.codehaus.groovy.m2eclipse 4.7.0.v202209301526-e2209-RELEASE) requires 'osgi.bundle; org.eclipse.m2e.core [1.9.1,2.0.0)' but it could not be found
  Cannot satisfy dependency:
    From: Groovy-Eclipse M2E integration 4.7.0.v202209301526-e2209-RELEASE (org.codehaus.groovy.m2eclipse.feature.feature.group 4.7.0.v202209301526-e2209-RELEASE)
    To: org.eclipse.equinox.p2.iu; org.codehaus.groovy.m2eclipse [4.7.0.v202209301526-e2209-RELEASE,4.7.0.v202209301526-e2209-RELEASE]
```

**Comment by eric-milles** [comment]
See #1392.  The range of compatible releases is indicated correctly at [1.9.1,2.0.0).  If you need to use Groovy m2e support, you can downgrade to the last m2e 1.x release using this update site: https://download.eclipse.org/technology/m2e/releases/1.20.1
